src/languageFeatures.ts

```diff
@@ -2,7 +2,7 @@ import { LanguageServiceDefaultsImpl } from './monaco.contribution';
 import { KustoWorker } from './kustoWorker';
 
 import * as ls from 'vscode-languageserver-types';
-import * as _ from 'lodash';
+import * as _ from 'lodash-amd';
```

Comment:
why did you need to change this?

Reply:
I thought I needed it through the PR cause I didn't find main.js under lodash package, turns out they have lodash.min so I reverted back to regular lodash